File: specification/specs/x_pack/async_search/get/AsyncSearchGetRequest.ts
@rest_spec_name("async_search.get")
class AsyncSearchGetRequest extends RequestBase {
  path_parts?: {
    id: string;
  }
  query_parameters?: {
  }
  body?: {
    keep_alive?: Time;
    typed_keys?: boolean;
    wait_for_completion_timeout?: Time;
  }
}

File: output/csharp/XPack/AsyncSearch/Get/AsyncSearchGetRequest.cs
using Nest.CommonOptions;
using Nest.CommonAbstractions;
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
namespace Nest.XPack {

	public class AsyncSearchGetRequest : RequestBase {
		
		[DataMember(Name="keep_alive")]
		public Time KeepAlive { get; set; }


		[DataMember(Name="typed_keys")]
		public bool TypedKeys { get; set; }


		[DataMember(Name="wait_for_completion_timeout")]
		public Time WaitForCompletionTimeout { get; set; }

	}
}
